packages/blocks/src/__internal__/utils/components.ts

```diff
@@ -74,7 +74,7 @@ function BlockElementWithService(
         state.then(() => {
           onLoaded();
         });
-        return html`<loader-element />`;
+        return html` <loader-element .hostModel=${model}> </loader-element> `;
```

Comment:
WebStorm will automatically add spaces in front of string templates, do you have any ideas on this?

Reply:
Nope